core/dbt/context/providers.py

```diff
@@ -1486,6 +1469,14 @@ def compiled_code(self) -> Optional[str]:
         else:
             return None
 
+    @contextproperty()
+    def sql(self) -> Optional[str]:
+        # only set this for sql models, for backward compatibility
+        if self.model.language == ModelLanguage.sql:  # type: ignore[union-attr]
+            return self.compiled_code
```

Comment:
If I'm following the logic correctly, I think this previously resulted in `self.model.compiled_code`. I want to make sure this is intentional since both are syntactically correct.